src/ngAnimate/module.js

```diff
@@ -267,9 +267,22 @@
  * .message.ng-enter-prepare {
  *   opacity: 0;
  * }
- *
  * ```
  *
+ * ### Animating between subsequent value changes
+ *
+ * Sometimes you need to animate between multiple (2 and more), different expression states, whose values
+ * don't necessary need to be known or referenced in CSS styles.
+ * Unless possible with more proper ["animation aware" directive](#directive-support), that specific
```

Comment:
I would remove "more proper". (What is "proper" anyway? Plus `ngAnimateSwap` is no less "proper" than other "animation aware" directives :grin:)
So, I would change _"with more proper"_ to _"with another"_.

Reply:
Some times a structural directive might be what you want (because for example you want to discard internal state :wink:). It depends on the usecase. (It is not that structural directives are objectively inferior to non-structural ones.)

Reply:
True